src/app.js

```diff
@@ -342,12 +349,228 @@ function buildImageImporter() {
 
   $importButton.click(function(){
     activateToolItem($importButton);
-    paper.initImageImport();
+    paper.initImageImport({mode:'trace'});
   });
 
   $('#tools #tool-select').before($importButton);
 }
 
+function buildImageVectorizer() {
+  var $importButton = $('<div>')
+      .addClass('tool')
+      .attr('id', 'importVectorized')
+      .data('cursor-key', 'select')
+      .attr('title', i18n.t('tracing.menutitle'));
+
+  var $imageDiv =  document.createElement("DIV");
+  $imageDiv.setAttribute("id", "imageDiv");
+  $imageDiv.style.backgroundImage = "url(images/icon-vectorize.png)";
+  $importButton.append($imageDiv);
+
+  // CREATE DROPDOWN MENU
+  var $dropdown =  document.createElement("DIV");
+  $dropdown.setAttribute("class", "dropdown-content");
+  $dropdown.setAttribute("id", "dropdownMenu");
+
+  // Levels title
+  var h1 = document.createElement("H3");
+  var t1 = document.createTextNode("Color Amount");     // Create a text node
+  h1.appendChild(t1);
+  $dropdown.appendChild(h1);
+
+  // Levels Slider
+  var sliderLevels = document.createElement("INPUT");
+  sliderLevels.setAttribute("type", "range");
+  sliderLevels.setAttribute("id", "sliderLevels");
+  sliderLevels.setAttribute("min", "2");
+  sliderLevels.setAttribute("max", "4");
+  sliderLevels.setAttribute("step", "1");
+  sliderLevels.setAttribute("value", "2");
+  $dropdown.appendChild(sliderLevels);
+
+  // Fidelity title
+  var h2 = document.createElement("H3");
+  var t2 = document.createTextNode("Edge Fidelity");     // Create a text node
+  h2.appendChild(t2);
+  $dropdown.appendChild(h2);
+
+  // Fidelity Slider
+  var sliderFidelity = document.createElement("INPUT");
+  sliderFidelity.setAttribute("type", "range");
+  sliderFidelity.setAttribute("id", "sliderFidelity");
+  sliderFidelity.setAttribute("min", "0");
+  sliderFidelity.setAttribute("max", "50");
+  sliderFidelity.setAttribute("step", "1");
+  sliderFidelity.setAttribute("value", "0");
+  $dropdown.appendChild(sliderFidelity);
+
+  // Area filter title
+  var h3 = document.createElement("H3");
+  var t3 = document.createTextNode("Minimal Area Filter");
```

Comment:
Again, as with the rest of these native document functions building out markup, we're using jQuery for these everywhere else, so these should be converted. Textnode data is set via element wrapper `.text('text content goes here')`.